lib/pure/parsecfg.nim

```diff
@@ -578,6 +580,11 @@ proc setSectionKey*(dict: var Config, section, key, value: string) =
   t[key] = value
   dict[section] = t
 
+iterator sections*(dict: Config): string {.since: (1, 3).} =
```

Comment:
Should probably use ``lent string`` as the return type.

Reply:
Am I missing something? Should i open an issue?

Reply:
Yes.

Reply:
see https://github.com/nim-lang/Nim/issues/17507

and workaround:
https://github.com/nim-lang/Nim/pull/17505